qri/cmd_util.py

```diff
@@ -39,5 +39,6 @@ def strip_color(colored_text):
 
 class QriClientError(RuntimeError):
     def __init__(self, err_string):
-        clean_err_string = strip_color(err_string.decode('utf-8'))
```

Comment:
This was failing to trigger the below error. It's expecting `bytes` from the `qri` CLI. (That's one reason I made it a special "CLI" error class).